### PR TITLE
Added the results of the new wordfreq_builder that splits emoji.

### PR DESCRIPTION
**Pull request by Joshua-Chin**

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -103,13 +105,13 @@ def test_phrase_freq():
...
-    eq_(random_words(nwords=4, lang='en', bits_per_word=0),
+    eq_(random_ascii_words(nwords=4, lang='en', bits_per_word=0),
         'the the the the')
 
     # This not only tests random_ascii_words, it makes sure we didn't end
     # up with 'eos' as a very common Japanese word
-    eq_(random_ascii_words(nwords=4, lang='ja', bits_per_word=0),
-        'rt rt rt rt')
+    eq_(random_words(nwords=4, lang='ja', bits_per_word=0),
+        'の の の の')
```
Comment (rspeer) [inline tests/test.py]: What's the reason for this change? The test doesn't match the comments anymore, and I'd hope that `random_ascii_words` isn't necessary to make "the" the most common English word.

Comment (Joshua-Chin) [inline tests/test.py]: The most common Japanese ascii word has changed almost every time we change wordfreq-builder ("html"->"rt"->"e")

Comment (rspeer) [inline tests/test.py]: The test still does ensure that we don't get "eos" (End Of Sentence) as a common Japanese word. This would actually happen with slightly incorrect uses of MeCab, and I think it's there in Leeds until we filter it out.
